manifests/h/HTTPToolKit/HTTPToolKit/1.4.1/HTTPToolKit.HTTPToolKit.locale.en-US.yaml

```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate
+
+PackageIdentifier: HTTPToolKit.HTTPToolKit
+PackageVersion: 1.4.1
+PackageLocale: en-US
+Publisher: HTTP Toolkit

```

Comment:
```suggestion
Publisher: Tim Perry
```

Reply:
@ItzLevvie Thanks for your suggestion.

I understand that Tim Perry is the one behind HTTP Toolkit. However, I believe he is using [HTTP Toolkit](https://github.com/httptoolkit) as the organization/publisher.

Also, in previous version [1.3.0 pull request](https://github.com/microsoft/winget-pkgs/pull/15425), I see that we have `Moniker` and `Publisher` value as `HTTP Toolkit`, so I keep the format. Will this change break anything?

I've read [Manifest Schema](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md) and I see that we have `Author` field that should match better. What do you think?

Reply:
This change was made to match it up with Control Panel; as this is what the Windows Package Manager uses to map applications in `winget list` and allow people to upgrade using `winget upgrade`.

If it needs to be changed to something else, you'll have to contact the application's developer for that.